src/peft/config.py

```diff
@@ -213,6 +214,21 @@ def _get_peft_type(
         loaded_attributes = cls.from_json_file(config_file)
         return loaded_attributes["peft_type"]
 
+    @classmethod
+    def check_kwargs(cls, **kwargs):
+        r"""
+        Check if the kwargs are valid for the configuration.
+
+        Args:
+            kwargs (additional keyword arguments, *optional*):
+                Additional keyword arguments passed along to the child class initialization.
+        """
+        if "oft_block_size" in kwargs:
```

Comment:
Hmm, so `oft_block_size` is the new parameter, right? So if we're loading an old OFT model, it should be missing. Therefore, should the check not be `if "oft_block_size" not in kwargs`?